packages/editor/src/components/post-title/index.js

```diff
@@ -123,7 +123,7 @@ class PostTitle extends Component {
 							<Textarea
 								id={ `post-title-${ instanceId }` }
 								className="editor-post-title__input"
-								value={ unescape( title ) }
+								value={ decodeEntities( title ) }
```

Comment:
I wonder a bit about the corresponding `escape` which happens in the `onUpdate` dispatch below. In truth, I'm not actually sure why it's needed at all. But at least previously, there was a duality between `unescape` and `escape` which is no longer guaranteed.

Reply:
Hmm I wonder why we added only decodeEntities in https://github.com/WordPress/gutenberg/pull/7977. I think an upstream patch to lodash (as including leading zeros is okay), and dependency update would work too, but I think I'm missing some context here.

Reply:
>I wonder a bit about the corresponding escape

Yes, it would be good to update that to an internal encoding method once one is available.